README.md

```diff
@@ -152,21 +152,9 @@ Tests can be run using [npm][npm]:
 
 ## Deployment
 
-When new code is pushed to master Travis CI runs a test suite against it. If those test
-pass [fabric][fabric] is used to deploy the new version to the server running
-under https://wo-ist-markt.de.
+Deployment is automatically done with Netlify. See the [contribution guide](CONTRIBUTING.md) for more details.
```

Comment:
I think this version is fine. It gives a broad introduction and defers the reader to where they can find more details. I'd leave it like this.